app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,68 @@
+class ArticlesController < ApplicationController
+  load_and_authorize_resource
```

Comment:
@sansuaza how the **load_and_authorize_resource** works?

Reply:
is a link to the ability class provided by device, this for the porpuse of load the access rules that we have put there